easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb

```diff
@@ -0,0 +1,326 @@
+easyblock = 'PythonBundle'
+
+name = 'QIIME2'
+version = '2023.7.0'
+
+homepage = 'https://qiime2.org'
+description = """QIIME 2 is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free,
+open source, and community developed."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+dependencies = [
```

Comment:
Add blank line between toolchain and dependencies

Reply:
fixed in 5496f24cd8